app/helpers/application_helper/page_layouts.rb

```diff
@@ -23,7 +24,7 @@ def layout_uses_tabs?
        (@layout == 'vm' && controller.action_name == 'edit') ||
        (@layout == "report" && ["new", "create", "edit", "copy", "update", "explorer"].include?(controller.action_name))
       return false
-    elsif @layout == "dashboard" # Dashboard tabs are located in taskbar because they are otherwise hidden behind the taskbar regardless of z-index -->
+    elsif @layout == "dashboard" || @layout == "container_dashboard" # Dashboard tabs are located in taskbar because they are otherwise hidden behind the taskbar regardless of z-index -->
```

Comment:
@zeari do we need to add this check here, i don't see any tabs on Container dashboard screen